src/libtriton/api/api.cpp

```diff
@@ -929,7 +929,33 @@ namespace triton {
     return this->symbolic->isOptimizationEnabled(opti);
   }
 
+  bool API::isMemorySymbolized(const triton::arch::MemoryAccess& mem) const{
+	  this->checkSymbolic();
+	  for (int i = 0; i < mem.getSize(); i++){
```

Comment:
`int i` must be `triton::uint32`. cf [travis](https://travis-ci.org/JonathanSalwan/Triton/jobs/168097209#L1184).